apps/nextjs-website/src/components/atoms/ProductHeader/ProductHeader.tsx

```diff
@@ -5,15 +5,13 @@ import { Box, useTheme } from '@mui/material';
 import { Header } from '@/editorialComponents/Header';
 import React, { FC } from 'react';
 import { useScrollUp } from '@/components/atoms/ProductHeader/useScrollUp';
+import { SITE_HEADER_HEIGHT } from '@/components/molecules/SiteHeader/SiteHeader';
```

Comment:
I'd have handled the constant in another way, maybe placed somewhere else, because in this way we are linking an atom, the `ProductHeader` with another component, the `SiteHeader`. I don't know if the atomic design has some best practice to handle case like that.

Anyway, the PR LGTM 👍